egs/iwslt22_ta/ST/local/compute_fbank_musan.py

```
@@ -0,0 +1 @@
+../../../librispeech/ASR/local/compute_fbank_musan.py
```

Ruff (0.12.2)

Shebang is present but file is not executable

(EXE001)